xmantissa/port.py

```diff
@@ -275,6 +275,68 @@ def listen(self):
 registerAttributeCopyingUpgrader(SSLPort, 1, 2)
 
 
+
+class EndpointPort(PortMixin, Item):
```

Comment:
Yes, in fact I had it named this way when I wrote #9, not sure where I lost the "String" part of the name. I've added it to the actual code now.